modules/apps/blogs/blogs-editor-configuration/src/main/java/com/liferay/blogs/editor/configuration/internal/BlogsCoverImageCaptionEditorConfigContributor.java

```diff
@@ -59,7 +59,7 @@ public void populateConfigJSONObject(
 			removePlugins = removePlugins + ",magicline,stylescombo,videoembed";
 		}
 		else {
-			removePlugins = "magicline";
+			removePlugins = "magicline,stylescombo,videoembed";
```

Comment:
`magicline,stylescombo,videoembed` should probably be in a variable, so we don't make the same mistake because of duplication.

Reply:
done!